libsolidity/analysis/ConstantEvaluator.cpp

```diff
@@ -255,6 +258,8 @@ optional<TypedRational> constantToTypedValue(Type const& _type)
 		return nullopt;
 }
 
+}
```

Comment:
Please add a comment denoting end of anon namespace for better readability

Reply:
Isn't it obvious? Do we use it in other places?

Reply:
> Do we use it in other places?

No

Reply:
I don't know if we use it elsewhere but it took me a second pass to ensure only the said functions were in the anonymous namespace and not others.